libs/ngrx-toolkit/src/lib/mutation/rx-mutation.ts

```diff
@@ -35,46 +35,70 @@ export interface RxMutationOptions<P, R> {
  *
  * The `operation` is the only mandatory option.
  *
+ * The returned mutation can be called as an async function and returns a Promise.
+ * This promise informs about whether the mutation was successful, failed, or aborted
+ * (due to switchMap or exhaustMap semantics).
+ *
+ * The mutation also provides several Signals such as error, status or isPending (see below).
+ *
+ * Example usage without Store:
+ *
  * ```typescript
- * export type Params = {
- *   value: number;
- * };
+ * private counterSignal = signal(0);
```

Comment:
The example reads like it is a class. Should we add the class to it or just remove the access modifier and go with simple `const`?

So:

```typescript
const counterSignal = signal(0);

const increment = rxMutation({
    // ...
```